plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ProgramConverter.kt

```diff
@@ -137,6 +157,47 @@ class ProgramConverter(val session: FirSession, override val config: PluginConfi
             }
     }
 
+    private fun processFunction(symbol: FirFunctionSymbol<*>, signature: FullNamedFunctionSignature): UserFunctionEmbedding =
+        UserFunctionEmbedding(processCallable(symbol, signature))
+
+    private fun processCallable(symbol: FirFunctionSymbol<*>, signature: FullNamedFunctionSignature): CallableEmbedding =
+        if (symbol.isInline) {
+            InlineNamedFunction(signature, symbol)
+        } else {
+            NonInlineNamedFunction(signature)
+        }
+
+    private fun convertMethodWithBody(declaration: FirSimpleFunction, signature: FullNamedFunctionSignature): Method {
+        val methodCtx = object : MethodConversionContext, ProgramConversionContext by this {
+            override val signature: FullNamedFunctionSignature = signature
+            override val nameMangler = NoopNameMangler
+            override fun getLambdaOrNull(name: Name): SubstitutionLambda? = null
+        }
```

Comment:
Nit: we can create the method context inside of the let since we don't need it otherwise.